README.md

```diff
@@ -46,6 +46,10 @@ For a local installation, make sure you have
 [pip installed](https://pip.readthedocs.io/en/stable/installing/) and run:
 
     $ pip install notebook
+             or
+    $ pip install jupyter
+             or
+    $ python -m pip install jupyter
```

Comment:
I don't think the `python -m` variant is necessary since it would also apply to the original `pip` statement as well.

Reply:
After reading up on when `python -m` should be used, and, again, since it would also apply to `pip install notebook`, I think converting the latter example into more of a comment, might be preferred, leaving something like:
```
$ pip install notebook
    or
$ pip install jupyter
```    
`Note: if multiple python environments are present, it is recommended that each of the above commands 
be prefixed with `python -m` (e.g., `python -m pip install notebook`) to ensure the python used for `pip` 
is the currently activated python.`